pkg/services/login/authinfoservice/service.go

```diff
@@ -25,7 +25,13 @@ func ProvideAuthInfoService(userProtectionService login.UserProtectionService, a
 		authInfoStore:         authInfoStore,
 		logger:                log.New("login.authinfo"),
 	}
-	usageStats.RegisterMetricsFunc(authInfoStore.CollectLoginStats)
+	// collect login stats
+	go func() {
```

Comment:
This causes usage statistics to not have the stats (checked using `/api/admin/usage-report-preview`).

`usageStats.RegisterMetricsFunc(authInfoStore.CollectLoginStats)` should still be called here and a background service registered for the prometheus stats

Reply:
> usageStats.RegisterMetricsFunc(authInfoStore.CollectLoginStats) should still be called here and a background service registered for the prometheus stats

I see, thanks for the thorough review to check the api/admin/usage-report endpoint.

Reply:
took back registerMetricsFunc